AUTHORS.rst

```diff
@@ -0,0 +1,8 @@
+rho was written by Adrian Likins <alikins@redhat.com>, Devan Goodwin
```

Comment:
Yes - he was part of the development of both rho 1.0 and 2.0, so it would be good to add him in